src/XMakeBuildEngine/BackEnd/BuildManager/BuildParameters.cs

```diff
@@ -320,6 +325,7 @@ private BuildParameters(BuildParameters other)
             _disableInProcNode = other._disableInProcNode;
             _logTaskInputs = other._logTaskInputs;
             _logInitialPropertiesAndItems = other._logInitialPropertiesAndItems;
+            _warningsAsErrors = other._warningsAsErrors == null ? null : new HashSet<string>(other._warningsAsErrors, StringComparer.OrdinalIgnoreCase);
```

Comment:
I did some limited research on what was faster and from what I could tell there was no difference.  Do you know of any guidance on which is faster?  I wanted the fastest possible lookup for sure.